docs/guides/alert-manager-configuration.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Configuring Notification using Cortex AlertManager"
+linkTitle: "AlertManager configuration"
+weight: 10
+slug: AlertManager-configuration
+---
+
+### Context
+
+Cortex AlertManager notification setup follow mostly the syntax of Prometheus AlertManager since it is based on the same codebase.  The following is a description on how to load the configuration setup so that AlertManager can use for notification when an alert event happened
+
+#### Cortex AlertManager configuration
+
+Cortex AlertManager can be uploaded via [Cortex Set Alertmanager API](https://cortexmetrics.io/docs/api/#set-alertmanager-configuration) or using Grafana Labs [Cortex Tools](https://github.com/grafana/cortex-tools)
+
+The following describe the step to load the configuration to Cortex AlertManager using `cortextool`.  Follow the instruction above to download or update to the latest version of `cortextool`.
+
+To obtain the full help of how to use `cortextool` for all commands and flags, use
+`cortextool --help-long`
+
+>  1. Create `.yaml` file to specify the configuration for AlertManager notification characteristic.  
+
+>>> The following is `amconfig.yml`, an example of a configuration for AlertManager to send notification to a Slack channel
```

Comment:
Let's remove `>>>`, and make it a normal text instead of triple-blockquote. Same for the text below.